pluginconfig/pluginconfig-311.yaml

```diff
@@ -137,7 +137,7 @@ versions:
       startup: quay.io/openshift-on-azure/azure:latest
       sync: quay.io/openshift-on-azure/azure:latest
       tlsProxy: quay.io/openshift-on-azure/azure:latest
-      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod07092019
+      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/cidev:redhat08242019
```

Comment:
Why we are using dev image here? Is this will be a supported path? What is the strategy to align image with generic insights image?

Reply:
@mjudeikis the `cidev:redhat08242019` image was provided by MSFT in order to debug some of the issues. The image contains some customizations done by them to achieve integration with ARO. Once everything is resolved, the changes will be ported into the official image. This is the reason this PR is currently on hold.

Reply:
@r-dilip says the new production image with the fixes will be provided within 2 days